### DISCUSSION TITLE
Getting errors when using pipewire and jack on linux

### DISCUSSION THREAD

**Discussion by kaanyalova** [ppy/osu#16893] [Q&A] [answered]
When I select `Jack Audio Connection Kit` as output source the game is constantly showing errors like these but everything else seems to work fine

![Ekran görüntüsü 2022-02-16 20-57-09](https://user-images.githubusercontent.com/76952012/154328303-e571894e-fc70-4183-b4d9-c8e416fd1d22.png)

[database.log](https://github.com/ppy/osu/files/8082387/database.log)
[network.log](https://github.com/ppy/osu/files/8082388/network.log)
[performance.log](https://github.com/ppy/osu/files/8082390/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8082391/runtime.log)

Os : Manjaro (Latest)
Game version: 2022.216.0-lazer (appimage)

**Comment by bdach** [comment]
i'm not sure why this has returned *again* but this seems to be a duplicate of https://github.com/ppy/osu/discussions/16179 somehow.

please post output of `LD_DEBUG=libs ./osu.AppImage`. as a text file if possible, it will be large.

**Reply by kaanyalova** [reply]
[output.txt](https://github.com/ppy/osu/files/8083197/output.txt)

**Reply by bdach** [reply]
to double-check, does `/usr/lib/bass.so` exist on your system? my suspicion is that the dynamic linker is choosing your system-wide BASS install over the binaries shipped with the appimage, and your system-wide version is older and has the bug that the binaries shipped with appimage don't have anymore (as it was fixed).

**Reply by kaanyalova** [reply]
`/usr/lib/bass.so` doesn't exist

**Reply by bdach** [reply]
not sure then, sorry. i can't reproduce this on my linux install so i can't debug. *if* bass has some special paths for jack/pipewire then that may be at fault, but i don't know if it does.

**Comment by kaanyalova** [comment, marked as answer]
seems to be fixed with the latest pipewire update
